src/net/github/Issues.fixture.js

```diff
@@ -1,5 +1,51 @@
 const GITHUB_BASE_URL = process.env.GITHUB_BASE_URL
 
+export const sampleIssues = [
+  {
+    id: 123,
+    title: 'issueTitle_1',
+    body: `issueBody_1. Lorem ipsum dolor sit amet, consectetur adipiscing
+    elit. Sed ac dolor sit amet purus malesuada congue. Nullam at arcu a
+    est sollicitudin euismod. Integer malesuada. Praesent blandit odio eu
+    enim. Pellentesque sed dui ut augue blandit sodales. Curabitur dictum
+    gravida mauris nam arcu libero, nonummy eget.
+    [bot-the-bldr image](https://github.com/OlegMoshkovich/Bldrs_Plaza/assets/3433606/1a6ecf3a-7422-4b8e-a895-2895051d53ae)
+    - [cam 1](http://localhost:8080/share/v/p/index.ifc#i:;c:-73.467,127.432,94,-27.69,7.071,1.331;)
```

Comment:
the host:port should be removed in the cards, so we should avoid setting them to something that looks meaningful

use e.g. http://bogus:0/ here and thruout

Reply:
ok

Reply:
ok